package.json

```diff
@@ -151,7 +151,7 @@
   "dependencies": {
     "JSONStream": "^1.3.5",
     "ansi-to-html": "^0.6.11",
-    "bson": "^4.0.2",
+    "bson": "github:addaleax/js-bson#after-the-next-bson-release-you-can-just-use-the-npm-package-again",
```

Comment:
Fwiw, this is the diff to v4.1.0:

https://github.com/addaleax/js-bson/commit/f07187a8a0428c3148c39893df18e650815fa16c